### PR TITLE
Drag n drop sorting for custom fields 

### PR DESCRIPTION
**Pull request by Kovah**
Companion pull request for https://github.com/bitwarden/desktop/pull/237

Screenshot 2019-03-27 at 15 22 24

Depends on the jslib pull request https://github.com/bitwarden/jslib/pull/35

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "test:watch": "karma start"
   },
   "devDependencies": {
-    "@angular/compiler-cli": "^7.2.1",
+    "@angular/compiler-cli": "^7.2.11",
```
Comment (kspearrin) [inline package.json]: please undo these changes for this PR since it is unrelated to the drag and drop feature.

File: package.json [hunk 2]
```diff
@@ -63,7 +63,7 @@
     "karma-jasmine": "^1.1.2",
     "karma-jasmine-html-reporter": "^1.3.1",
     "karma-typescript": "^3.0.13",
-    "node-sass": "^4.9.3",
+    "node-sass": "^4.11.0",
```
Comment (kspearrin) [inline package.json]: please undo these changes for this PR since it is unrelated to the drag and drop feature.

Comment (Kovah) [reply package.json]: Done.

Comment (kspearrin) [reply package.json]: This one still showing changed
